modules/currency-l0/src/main/scala/org/tessellation/currency/l0/modules/Storages.scala

```diff
@@ -2,42 +2,61 @@ package org.tessellation.currency.l0.modules
 
 import cats.effect.kernel.Async
 import cats.effect.std.{Random, Supervisor}
-import cats.syntax.flatMap._
-import cats.syntax.functor._
+import cats.syntax.all._
 
+import org.tessellation.currency.dataApplication.BaseDataApplicationL0Service
+import org.tessellation.currency.dataApplication.storage.CalculatedStateLocalFileSystemStorage
 import org.tessellation.currency.l0.node.IdentifierStorage
 import org.tessellation.currency.l0.snapshot.storages.LastBinaryHashStorage
+import org.tessellation.currency.schema.currency
 import org.tessellation.currency.schema.currency.{CurrencyIncrementalSnapshot, CurrencySnapshotInfo}
 import org.tessellation.kryo.KryoSerializer
 import org.tessellation.schema.peer.L0Peer
-import org.tessellation.schema.{GlobalIncrementalSnapshot, GlobalSnapshotInfo}
+import org.tessellation.schema.{GlobalIncrementalSnapshot, GlobalSnapshotInfo, SnapshotOrdinal}
 import org.tessellation.sdk.config.types.SnapshotConfig
 import org.tessellation.sdk.domain.cluster.storage.{ClusterStorage, L0ClusterStorage, SessionStorage}
 import org.tessellation.sdk.domain.collateral.LatestBalances
 import org.tessellation.sdk.domain.node.NodeStorage
 import org.tessellation.sdk.domain.snapshot.storage.{LastSnapshotStorage, SnapshotStorage}
 import org.tessellation.sdk.infrastructure.cluster.storage.L0ClusterStorage
 import org.tessellation.sdk.infrastructure.gossip.RumorStorage
-import org.tessellation.sdk.infrastructure.snapshot.storage.{LastSnapshotStorage, SnapshotLocalFileSystemStorage, SnapshotStorage}
+import org.tessellation.sdk.infrastructure.snapshot.storage._
 import org.tessellation.sdk.modules.SdkStorages
 
+import fs2.io.file.Path
+
 object Storages {
 
+  def dataApplicationCalculatedStatePath = Path("data/calculated_state")
```

Comment:
Shouldn't it be hardcoded in the config?

Reply:
It may be configurable but we would have to make it dependant on the data application part which isn't so obvious (require only if the data application implementation is provided). Any idea?